test/fixtures/component-components/parse-out.default

```diff
@@ -2,13 +2,16 @@
   {
     "elements": [
       {
-        "name": "Business Logic"
+        "name": "Business Logic",
+        "title": "Business Logic"
       },
       {
-        "name": "Data Access"
+        "name": "Data Access",
```

Comment:
should be DA?

```
[Data Access] as DA
```

Reply:
you are right, I forgot to implement the title feature for the short syntax for components/use-cases. Will fix.

Reply:
this should be fixed now.